src/runmode-af-packet.c

```diff
@@ -367,19 +367,34 @@ static void *ParseAFPConfig(const char *iface)
         SCLogDebug("Going to use cluster-id %" PRIu16, aconf->cluster_id);
     }
 
-    if (ConfGetChildValueWithDefault(if_root, if_default, "cluster-type", &tmpctype) != 1) {
+    int conf_val = 0;
+    bool rollover = false;
+    ConfGetChildValueBoolWithDefault(if_root, if_default, "rollover", &conf_val);
+    if (conf_val) {
+        SCLogWarning("%s: rollover deprecated; using \"cluster_flow\" instead. See "
+                     "https://redmine.openinfosecfoundation.org/issues/6128",
+                aconf->iface);
+        rollover = true;
+    }
+    if (rollover ||
```

Comment:
this confuses rollover and cluster_rollover. If just rollover was set, the cluster setting shouldn't be modified I think

Reply:
Specifying `rollover` or `cluster_rollover` results in the same cluster type flag -- `PACKET_FANOUT_ROLLOVER` to be used.

Do we want to remove `cluster_rollover` support but allow the other way of achieving the same result?

Reply:
I see `rollover` leading to the flag `PACKET_FANOUT_FLAG_ROLLOVER`, while `cluster_rollover` has cluster_type `PACKET_FANOUT_ROLLOVER`. The former is a flag in addition to the cluster type, the latter sets the cluster type.

Reply:
Understood; I'll remove the changes for `rollover`